app/providers/tripadvisor.py

```diff
@@ -15,11 +15,14 @@ def getNumId(idStr):
 params = { "key": tripadvisorkey }
 
 def resolve(idObj):
```

Comment:
nit: as in irl -> `resolveWithID(id)`

fwiw, it's `yelp.resolve_with_key` in yelp.